attic/rfc_reduction.md

```diff
@@ -0,0 +1,71 @@
+
+```
+ Feature Name: `Forecasting Reduction Techniques`
+ Start Date: 2019-11-16
+ Target Date:
+ ```
+
+## Summary
+[summary]: #summary
+ 
+ - Lagged values of a timeseries can be used as features for a machine learning algorithm.
+ - Two major apporaches: rolling window regression and expanding window regression
```

Comment:
Sorry if we are not talking about CV then idt there is such a thing as rolling and expanding window regression. Those are methods for validating time series.

Tuning the amount of data you show a model seems, as a hyperparameter for performance, very weird and dangerous to me

Reply:
This RFC is about giving a basic idea of how to rephrase forecasting problems to ordianry regression problems. 
Sorry, I confused some things about the two methods. I will update the rfc soon.